src/IdBrokerClient.php

```diff
@@ -717,9 +717,12 @@ public function updateUser(array $config = [ ]): array
     private function assertTrustedBrokerIp()
     {
         $baseHost = parse_url($this->idBrokerUri, PHP_URL_HOST);
-        $idBrokerIp = gethostbyname(
-            $baseHost
-        );
+        if ($baseHost == ($idBrokerIp = gethostbyname($baseHost))) {
```

Comment:
I would prefer having this split out ...
$idBrokerIp = ...;
if ($baseHost == $idBrokerIp) { ... }